types/session_price.yaml

```diff
@@ -0,0 +1,73 @@
+description: Types to provide the session price
+types:
+  SessionCostChunk:
+    description: >-
+      A chunk of the session cost. The total cost of the session is the sum of
+      all chunks.
+    type: object
+    additionalProperties: false
+    properties:
+      timestamp_from:
+        description: >-
+          Absolute timestamp for the start of this chunk in RFC3339 UTC format
+        type: string
+        format: date-time
+      timestamp_to:
+        description: >-
+          Absolute timestamp for the end of this chunk in RFC3339 UTC format
+        type: string
+        format: date-time
+      cost_energy:
```

Comment:
I am a fan of describing currency as two integers, namely the value and the exponent. For example if you want to specify 10 eurocent, you could do it as { "EUR", 10, -2 }. The equivalent int bitcoin would be { "BitCoin", 3, -5 }. I learned that at Lyft where all currency are represented like that and in fact we do the same in the Qwello backend.

The advantage of course is you work around rounding errors and you can work with fractions of the actual currency safely. I suggest using this here too.